### ISSUE TITLE
Document Amazon CloudWatch support

### ISSUE THREAD

**Issue by tom-mi** [micrometer-metrics/micrometer-docs#256]
Hello,

there are two implementations for cloudwatch registries [micrometer-registry-cloudwatch](https://github.com/micrometer-metrics/micrometer/tree/main/implementations/micrometer-registry-cloudwatch) / [micrometer-registry-cloudwatch2](https://github.com/micrometer-metrics/micrometer/tree/main/implementations/micrometer-registry-cloudwatch2), but there is no documentation how to use it (at least there seems to be no link / way to find it on https://micrometer.io/docs )

Please fix :)
Thx

**Comment by shakuzen** [comment]
Thank you for opening the issue. It may be the case that your browser is caching an old version of the site. We used to have some issues of things being cached for way too long due to service workers. We have since fixed that, but it requires clearing browser cache once since then.
We have https://micrometer.io/docs/registry/cloudwatch which is linked from https://micrometer.io/docs in the latest version of the live site.

**Comment by tom-mi** [comment]
Yeah you're right, opening it in another browser works, clearing the cache also does the trick.
Thanks for the quick response!
